step-5/README.md

```diff
@@ -124,33 +135,36 @@ Create the function in your `methods: {..}` object, like this:
   },
 },
 ```
-Notice that the code above is almost an identical copy of the code from your `script.js` file, *_except_* that any mention of a formerly-global variable has been update to be prefixed with `this.`. That's how the Vue app knows that the variables in question are scoped to _*this*_ component.
+
+Notice that the code above is almost an identical copy of the code from your `script.js` file, _*except*_ that any mention of a formerly-global variable has been update to be prefixed with `this.`. That's how the Vue app knows that the variables in question are scoped to _*this*_ component.
 
 You may also see a syntax error under the `matchCards()` function call - that's because you have not yet defined that function!
 
 Before you get to the `matchCards()` function, you will want to add the `flipCard` function as a click event callback. With Vue, this is easy...
 
 In your `GameBoard.vue` component, find your `<li>` tag:
+
 ```html
-<li v-for="(cardInfo, index) in cardsData" :key="index" class="card">
+<li v-for="(cardInfo, index) in cardsData" :key="index" class="card"></li>
 ```
+
 Reformat this tag so that the attributes/props/directives all appear stacked:
+
 ```html
-<li
-  v-for="(cardInfo, index) in cardsData"
-  :key="index"
-  class="card"
->
+<li v-for="(cardInfo, index) in cardsData" :key="index" class="card"></li>
 ```
-Then add a _vue-bound click event handler_ on the line following the `class` attribute, like this:
+
+Then add a [_vue-bound click event handler_](https://v2.vuejs.org/v2/guide/events.html) on the line following the `class` attribute, like this:
+
 ```html
 <li
   v-for="(cardInfo, index) in cardsData"
   :key="index"
   class="card"
   @click="flipCard"
->
+></li>
```

Comment:
Please remove this closing tag as well. Student work at this point in the assignment should have active code inside the `li` tag so I think it would be confusing to have this tag closed without content in the example - especially if this is being copied & pasted - the closing tag would break the assignment code.